docs/frontend/in-depth/i18n.md

```diff
@@ -0,0 +1,102 @@
+# i18n interface translations
+
+The i18n mechanism is a library that allows you to easily add interface translations.<br>
```

Comment:
The i18n mechanism is a library that allows you to add interface translations more easily.